student.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, classroom, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] Initializing classroom is not needed in this case because the initialize method sets the classroom instance variable, but it does not add the student to the student's array of the classroom. This means that if a student is created using this constructor, they will not be associated with any classroom until the ```classroom``` method is called separately as you did below.

Reply:
resolved! ✔️